eth_dummy_phy/idf_component.yml

```diff
@@ -0,0 +1,9 @@
+version: 0.5.0
+targets:
+  - esp32
+  - esp32p4
+description: This component creates dummy instance of Ethernet PHY and so two devices can be connected directly by EMAC to EMAC.
+url: https://github.com/espressif/esp-eth-drivers/tree/master/eth_dummy_phy
+dependencies:
+  idf:
+    version: '>=5.3'
```

Comment:
question, can the dummy PHY work with earlier IDF release version?

Reply:
Theoretically yes, but it is dependent on [this](https://gitlab.espressif.cn:6688/espressif/esp-idf/-/merge_requests/29058) update. It was backported back to 5.0. However, I cannot easily define IDF version in manifest file since the fix is available only from specific patch version for each IDF minor version. It would have to look like `~5.0.7 || ~5.1.4 || ~5.2.2 || >=5.3` which is not possible. Or I would have to release multiple component release version which could be maintenance nightmare.

The simplest solution is to make it compatible since v5.2.2...